src/Compilers/CSharp/Portable/Symbols/Source/SourcePropertySymbolBase.cs

```diff
@@ -783,6 +783,11 @@ internal bool IsAutoProperty
                 return null;
             }
         }
+
+        internal bool GetMethodIsEquivalentToBackingFieldRead => _getMethod?.BodyShouldBeSynthesized == true;
+
+        internal bool SetMethodIsEquivalentToBackingFieldWrite => _setMethod?.BodyShouldBeSynthesized == true;
```

Comment:
>SetMethodIsEquivalentToBackingFieldWrite [](http://example.com/codeflow?start=22&length=40)

Similar comment. We will have one API named something like "EquivalentToToBackingFieldAccess" instead of two. Also, we wouldn't need to worry about what should be returned for an accessor that doesn't exist.